.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
```

Comment:
Is it required?

Reply:
I just included the .gitattributes file to fix the line endings formatting but forgot to add the file to the .gitignore.